test/functional/ui/decorations_spec.lua

```diff
@@ -1972,30 +2404,42 @@ describe('extmark decorations', function()
     ]])
 
     -- When only one highlight group has an underline attribute, it should always take effect.
-    for _, d in ipairs({-5, 5}) do
+    for _, d in ipairs({ -5, 5 }) do
       api.nvim_buf_clear_namespace(0, ns, 0, -1)
       screen:expect([[
         aaabbbaa^a                                         |
         {0:~                                                 }|
                                                           |
       ]])
       api.nvim_buf_set_extmark(0, ns, 0, 0, { end_col = 9, hl_group = 'TestUL', priority = 25 + d })
-      api.nvim_buf_set_extmark(0, ns, 0, 3, { end_col = 6, hl_group = 'TestBold', priority = 25 - d })
+      api.nvim_buf_set_extmark(
```

Comment:
As a modest proposal, could we perhaps increase the column threshold from 100 to like 120? 

This would avoid a lot of _deformatting_ like this. Nothing about this line is different from the line just above. It shouldn't have a completely different style just because a string was one single ASCII byte longer.

Reply:
Alternatively, we could actively poke Stylua about it; there's still development activity.

Reply:
> As a modest proposal, could we perhaps increase the column threshold from 100 to like 120?

Pushed an update.

Are we ok with the changes to other test files, or should the 120 width be limited to certain files (idk if that's possible)